### ISSUE TITLE
Release not detected

### ISSUE THREAD

**Issue by serphacker** [sibbell/support#77]
Example: 

https://sibbell.com/github/swimlane/ngx-datatable/releases/ latest release = 4.2.0 1 month, 1 week ago

However, latest release is 6.3.0

https://github.com/swimlane/ngx-datatable/releases

**Comment by davegaeddert** [comment]
Hey @serphacker, thanks for pointing it out. So on GitHub it shows both git tags and published "GitHub Releases". The way that Sibbell works currently, if the repo has ever published actual "releases", then we only look for those. That way they can push git tags around in development and whenever they're ready to publish them as "releases", that's when they show up in Sibbell. In this case though, it looks like they used the publishing process before but no longer do... Not quite sure what the best answer is at the moment. Thoughts?

**Comment by martinml** [comment]
[This project](https://github.com/urandom/gearshift/releases) seems to be doing releases but they are not appearing in Sibbell for me. Is it the same problem or am I misunderstanding what a proper "GitHub release" is? 🤔

**Comment by davegaeddert** [comment]
@martinml just looking at this now -- do see the releases listed here https://sibbell.com/github/urandom/gearshift/releases/? Looks correct to me but maybe it's been updated since you asked...

**Comment by martinml** [comment]
Yes! They do appear now. Probably I was too impatient and they were still being fetched when I looked. Thanks for the reply.

**Comment by freMea** [comment]
Same issue goes for [https://github.com/Microsoft/vscode/releases](https://github.com/Microsoft/vscode/releases)

This needs to be fixed or this tool looses interest.
